metricbeat/module/elasticsearch/cluster_stats/data_xpack.go

```diff
@@ -142,6 +142,22 @@ func apmIndicesExist(clusterState common.MapStr) (bool, error) {
 	return false, nil
 }
 
+func getClusterMetadataSettings(m *MetricSet) (common.MapStr, error) {
+	// For security reasons we only get the display_name setting
```

Comment:
Why is that?

Reply:
Users can set whatever they want in the cluster metadata. They might set sensitive information, and we would end up indexing it into monitoring indices. So, to be on the safer side, we decided to just collect the specific key, `display_name`, from cluster metadata for now.

See complete discussion about this starting here: https://github.com/elastic/elasticsearch/pull/34023#issuecomment-424697615.

Note that even though we only collect the specific key for now, we are indexing the complete nesting structure for the key. That way, if we later decide to allow more keys or all keys, there will be no break in the expected structure for the UI.